book/modules/appendix1.tex

```diff
@@ -351,24 +388,201 @@
 	\end{enumerate}
 
 	\begin{example}
-		XXX Finish. Unique solution to a 3x3 system with no row swaps; Use back subsitution
+		Find a solution to the following system:
+		\[
+			\systeme{
+				a+3b+2c=1,
+				2a+7b+5c=2,
+				-a-4b=11
+			}.
+		\]
+		To do so, we rewrite the system as an augmented matrix then row reduce.
+		\begin{align*}
+			{\footnotesize
+			\systeme{
+				a+3b+2c=1,
+				2a+7b+5c=2,
+				-a-4b=11
+			}
+			} \rightarrow
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				2&7&5 & 2\\
+				-1&-4&0 & 11
+			\end{bmatrix}
+			&\xrightarrow{\text{row}_2\mapsto\text{row}_2-2\text{row}_1}
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				0&1&1 & 0\\
+				-1&-4&0 & 11
+			\end{bmatrix}\\
+			&\xrightarrow{\text{row}_3\mapsto\text{row}_3+\text{row}_1+\text{row}_2}
+			\begin{bmatrix}[rrr|r]
+				1&3&2 & 1\\
+				0&1&1 & 0\\
+				0&0&3 & 12
+			\end{bmatrix}
+			\rightarrow
+			{\footnotesize
+				\systeme{
+				a+3b+2c=1,
+				b+c=0,
+				3c=12
+				}
+			}
+		\end{align*}
+		The third row reveals that $c=4$; substituting into the second row, we find $b=-4$. Now we can substitute $b=-4$ and $c=4$ into the first row and we obtain $a=5$.
+		
+		The solution is $\mat{a\\b\\c}=\mat{5\\-4\\4}$. Since this is the only solution to the system, the solution set is $\Set*{\mat{5\\-4\\4}}$.
 	\end{example}
 
 	\begin{example}
-		XXX Finish. Unique solution to a 3x4 system that needs a row swap; Use back subsitution
+		Solve the system
+		\[
+			\systeme[tsr]{
+				3t+s+13r=-2,
+				t+5r=1,
+				-t+s-6r=-8,
+				t+s+4r=-6
+			}.
+		\]
+		Again, we row reduce the corresponding augmented matrix to find an equivalent system\index{System of linear equations!equivalent} that is easier to find the solution set.
```

Comment:
Huh? Maybe "find an equivalent system from which we can more easily compute the solution"